Makefile

```diff
@@ -260,13 +276,12 @@ bundle-community-k8s: bundle-community ## Generate bundle manifests and metadata
 	sed -r -i "s|by default\.|by default.\n    Note that prior to installing SNR on a Kubernetes 1.25+ cluster, a user must manually set a [privileged PSA label](https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/) on SNR's namespace. It gives SNR's agents permissions to reboot the node (in case it needs to be remediated).|;" ./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml
 
 .PHONY: bundle-community
-bundle-community: bundle
-	#Add Community Edition suffix to operator name
+bundle-community: bundle ##Add Community Edition suffix to operator name
 	sed -r -i "s|displayName: Self Node Remediation Operator.*|displayName: Self Node Remediation Operator - Community Edition|;" ./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml
 
 .PHONY: bundle-update
-bundle-update:
-    # update container image in the metadata
+bundle-update: ## Update containerImage, createdAt, skipRange, and icon fields in the bundle's CSV, then validate the bundle directory
```

Comment:
I think the comment is wrong, IIUC it only updates  containerImage & createdAt,

Reply:
In [lines](https://github.com/medik8s/self-node-remediation/pull/122/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R288-R291) it does update  skipRange, and icon fields.